src/shims/unix/fs.rs

```diff
@@ -13,10 +13,17 @@ use rustc_data_structures::fx::FxHashMap;
 use rustc_middle::ty::{self, layout::LayoutOf};
 use rustc_target::abi::{Align, Size};
 
+use crate::shims::os_str::bytes_to_os_str;
 use crate::*;
 use shims::os_str::os_str_to_bytes;
 use shims::time::system_time_to_duration;
 
+const TEMPFILE_TEMPLATE_STR: &str = "XXXXXX";
+// <https://github.com/lattera/glibc/blob/895ef79e04a953cac1493863bcae29ad85657ee1/sysdeps/posix/tempname.c#L203>
+const TEMPFILE_ATTEMPTS_MIN: u32 = 62 * 62 * 62;
+// <https://github.com/lattera/glibc/blob/895ef79e04a953cac1493863bcae29ad85657ee1/sysdeps/posix/tempname.c#L36>
+const TEMPFILE_ATTEMPTS_GLIBC: u32 = 238328;
```

Comment:
Please make these local constants in the implementation.

Also, do we *want* to implement this glib-specific option? Seems better to just do the minimal number of attempts that is required by the spec.

Reply:
I changed it so we only do the minimum.